tests/unit/net/data_loader.rs

```diff
@@ -47,7 +47,10 @@ fn empty_invalid() {
 
 #[test]
 fn plain() {
-    assert_parse("data:,hello%20world", None, None, Some(b"hello world".iter().map(|&x| x).collect()));
+    assert_parse(
+        "data:,hello%20world",
+        Some(ContentType(Mime(TopLevel::Text, SubLevel::Plain, vec!()))),
```

Comment:
This should be `Some(ContentType(Mime(TopLevel::Text, SubLevel::Plain, vec!((Attr::Charset, Value::Ext("US-ASCII".to_owned()))))),` or something like that.